showcase.json

```diff
@@ -907,6 +907,220 @@
       "/img/showcase/astrox_me_screenshot.webp"
     ]
   },
+  {
+    "id": "funded",
+    "name": "Funded",
+    "onLiner": "Crowdfund the Future! Using smart contract technology to build a safer, cheaper and global crowdfunding service - with a new NFT model.",
+    "tags": [
+      "SocialFi",
+      "Tools / Infrastructure"
+    ],
+    "useInternetIdentity": true,
+    "website": "https://funded.app/",
+    "github": "https://github.com/CrowdFund-NFT/escrow_manager",
+    "twitter": "https://twitter.com/ApocStudiosInc?s=20&t=jqHbKZjRS47NdOcbDHfK6w",
+    "discord": "https://discord.gg/thqARE5JqF",
+    "logo": "/img/showcase/funded_logo.webp",
+     "submittableId": "26383175"
+  },
+  {
+    "id": "argonstudio",
+    "name": "Argon Studio",
+    "onLiner": "We'll be your end-to-end partner, whether managing your project from ideation to implementation, or filling gaps in your current team.",
```

Comment:
Fails to display: 
<img width="349" alt="image" src="https://user-images.githubusercontent.com/14757975/229052646-8529a194-ce30-4fc3-be7a-6b9076b4d403.png">